crates/sui-benchmark/src/drivers/bench_driver.rs

```diff
@@ -532,7 +589,52 @@ impl Driver<BenchmarkStats> for BenchDriver {
             }
             res = all_tasks => res.unwrap().into_iter().collect()
         };
-        let benchmark_stat = stat_task.await.unwrap();
-        Ok(benchmark_stat)
+        let benchmark_stat = benchmark_stat_task.await.unwrap();
+        let stress_stat = stress_stat_task.await.unwrap();
+        Ok((benchmark_stat, stress_stat))
     }
 }
+
+fn stress_stats_collector(
```

Comment:
Maybe have a high level description of how this works, e.g. collection interval, when will this end, etc